StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/RemoveButton.swift

```diff
@@ -43,16 +43,10 @@ class RemoveButton: UIButton {
         titleLabel?.textAlignment = .center
         titleLabel?.font = font
         titleLabel?.adjustsFontForContentSizeCategory = true
-        let strokeWidth = appearance.selectedBorderWidth ?? appearance.borderWidth * 1.5
-        if LiquidGlassDetector.isEnabled {
-            ios26_applyCapsuleCornerConfiguration()
-            // The UI breaks if stroke with <= 0, so default to 1.5. Follows pattern from LiquidGlassRectangle
-            // Workaround:  Use layer.borderWidth because background.strokeWidth isn't compatible with .capsule()
-            layer.borderWidth = strokeWidth > 0 ? strokeWidth : 1.5
-        } else {
-            layer.borderWidth = strokeWidth
-            layer.cornerRadius = appearance.primaryButton.cornerRadius ?? appearance.cornerRadius
-        }
+        applyCornerRadius(appearance: appearance, ios26DefaultCornerStyle: .capsule, shouldUsePrimaryButtonCornerRadius: true)
+        // Workaround:  Use layer.borderWidth because background.strokeWidth isn't compatible with .capsule()
+        layer.borderWidth = 1.5
+
```

Comment:
cc @gbirch-stripe - I don't think this should use `appearance` for its outline border width so I changed it to always be 1.5